src/runtime/local/kernels/CMakeLists.txt

```diff
@@ -118,3 +210,6 @@ find_library(PAPI_LIB NAMES libpapi.so HINTS ${PROJECT_BINARY_DIR}/installed/lib
 find_library(HWLOC_LIB NAMES libhwloc.so HINTS ${PROJECT_BINARY_DIR}/installed/lib REQUIRED)
 
 target_link_libraries(AllKernels PUBLIC ${LIBS} ${MPI_LIBRARIES} ${PAPI_LIB} ${HWLOC_LIB})
+
+# Add SIMDOperators
+add_subdirectory(SIMDOperatorsDAPHNE)
```

Comment:
Maybe think of another name for this? It's the DAPHNE source repository - no need to add DAPHNE in the directory name here. Also SIMDOperators is a bit generic - the operators using OpenBLAS would also qualify for that I think ;-)